docs/en/guides/How-to-build.md

```diff
@@ -133,6 +133,6 @@ Refer to [Build docker image](../../../docker) for more details.
 </dependency>
 ```
 5. Run `./mvnw compile -Dmaven.test.skip=true`
-6. Run `maven update`. Must remove the clean projects item before maven update(This will be clear the proto conversion Java file generated by the complie)
+6. Run `./mvnw update`. Must remove the clean projects item before maven update(This will be clear the proto conversion Java file generated by the compile)
```

Comment:
`./mvnw update` is not a command.

Reply:
may be `./mvnw clean` ?

Reply:
To be honest, I am not sure what does this part mean. As I never use Eclipse these days.

Reply:
> `./mvnw update` is not a command.

Got it. It's a feature only on Eclipse.

Reply:
If so, We should keep it without chenges. :P